metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -45,6 +45,12 @@ under +/var/lib/{beatname_lc}-data+. The manifest uses folder autocreation (`Dir
 Kubernetes 1.8. You need to remove `type: DirectoryOrCreate` from the manifest and create the host folder yourself.
 =======================================
 
+[WARNING]
+=======================================
+*You need to ensure that your indices are generated by the same version of Metricbeat that is running in the cluster. If you do have errors in your container, you should check permissions and you may want to try using the latest update.
```

Comment:
The phrase `You need to ensure that your indices are generated by the same version of Metricbeat that is running in the cluster.` looks fine.

The rest part is too generic. What kind of permissions we are referring to? You might mean RBAC permissions? Maybe we could be more specific by adding specific an example.

Also the note about using the latest update seems to be redundant? This specific documentation already suggests downloading this manifest (with the corresponding version taken into account) in the beginning.